src/components/TodoAdd.tsx

```diff
@@ -0,0 +1,25 @@
+type Props = {
+  callback: (todoTitle: string) => void;
+}
+
+function TodoAdd(props: Props) {
+  const addTodo = ($event: any) => {
+    $event.preventDefault();
+    const value = $event.target[0].value;
+    $event.target[0].value = null;
```

Comment:
Try to use a state for the input. It’s not a good practice to change something in dom directly

Reply:
done